src/style.css

```diff
@@ -0,0 +1,86 @@
+:root {
+  --grey: #e1e7ed;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Heavy.ttf");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf");
+  font-weight: bold;
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.form-item {
+  width: 100%;
+  height: 70px;
+  font-weight: 300;
+  border: 1px solid var(--grey);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  transition: 3s;
+  line-height: 22px;
+  padding: 25px 0 23px 62px;
+}
+
+.form-item::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.form-small-item::placeholder {
+  font-size: 14px;
+}
+
+.form-img {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  height: 19px;
+  width: 19px;
+}
+
+.form-small-item {
+  height: 42px;
+  font-size: 14px;
+  padding: 12px 0 10px 33px;
+}
+
+.form-small-img {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.form-item:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-item:focus {
+  border: 1px solid var(--grey);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
```

Comment:
move to a variable

Reply:
Not fixed, please move it to css variable